pkg/util/k8s/node.go

```diff
@@ -65,14 +82,17 @@ func IsPodRecentlyCreatedAfterNodeCordoned(
 		return true
 	}
 
-	lastCreationTime, ok := lastCreationTimeOnNode[node.Name]
-	// If the pod has never been created, no timestamp.
-	if !ok {
-		return false
+	// If pod has been recently created, return true
+	// Otherwise the pod will get deleted, increase the restart delay
+	recentlyCreated := cutOffTime.Before(nodeInfo.LastPodCreationTime)
+	if !recentlyCreated && !overwriteRestartDelay {
+		nodeInfo.CordonedRestartDelay = waitDuration * 2
+		if nodeInfo.CordonedRestartDelay > constants.MaxCordonedRestartDelay {
+			nodeInfo.CordonedRestartDelay = constants.MaxCordonedRestartDelay
+		}
+		logrus.Infof("node %s is cordoned, next pod restart is scheduled in %s", nodeInfo.NodeName, nodeInfo.CordonedRestartDelay)
```

Comment:
"in %s" -> "in %d seconds"

Reply:
it's already a time.duration object, so it will be parsed correctly, with %d, it will become nanosecond